qdarkstyle/qss/_styles.scss

```diff
@@ -1194,7 +1194,7 @@ QComboBox {
     border-radius: $SIZE_BORDER_RADIUS;
     selection-background-color: $COLOR_SELECTION_NORMAL;
     padding-left: 4px;
-    padding-right: 4px;
+    padding-right: 36px;  /* 4 + 16*2 See scrollbar size */
```

Comment:
Ok, just that? :)

Reply:
Yes, basically we need to add enough padding to account for the scrollbar size 16*2 + the original padding (4)

So yeah :-)